### ISSUE TITLE
Suggestion / request: use Dependabot to keep dependencies up-to-date

### ISSUE THREAD

**Issue by greysteil** [SUSE/Portus#1994]
First of all, thanks for Portus!

I've got a suggestion / request: would you be up for using [Dependabot](https://dependabot.com) to automatically create dependency update PRs for this repo? I ran it against my fork and it generated [these PRs](https://github.com/greysteil/Portus/pulls). I'll port the `nokogiri` one across to this repo now because it's security related.

I built Dependabot, but I'm honestly only suggesting it because I hope it can save you some time. I'd love any feedback, and obviously having open source repos using Dependabot helps boost its profile, but if it's not helpful to you then it's not really worth anything.

You can install it from [here](https://github.com/apps/dependabot) or [here](https://github.com/marketplace/dependabot) if you decide to give it a try. It's been through GitHub's security testing (to be allowed in the GitHub Marketplace) and is used by a few thousand organisations, and the source code is [here](https://github.com/dependabot/dependabot-core).

I'm very happy to pitch in and help with the initial dependency update.

:octocat:

**Comment by mssola** [comment]
A couple of questions before considering further:

- Is there a way to tell dependabot to only check for security issues? (as in, with CVE).
- Is there a way to tell dependabot to only check specific branches? In my case, the master branch doesn't need all this considering our workflow.
- Is there a way to tune the commit message?

Thanks!

**Comment by greysteil** [comment]
> Is there a way to tell dependabot to only check for security issues? (as in, with CVE).

Yes - when you add a project to Dependabot you can set it to only create PRs for security vulnerabilities (it's in the "advanced options" section).

> Is there a way to tell dependabot to only check specific branches? In my case, the master branch doesn't need all this considering our workflow.

Yes - you can choose which branch Dependabot targets when you add a project to it (again it's in the "advanced options" section).

> Is there a way to tune the commit message?

Not at the moment - the commit messages would look like [this one](https://github.com/SUSE/Portus/pull/1995/commits/20bfcd3fb71565e88f4eda0f9977f35e9ef1b44a). It will automatically detect whether you're using semantic commit messages, but it doesn't look like it thinks you are. Is there a particular style you're looking for? Note that is *does* sign its work as suggested in the contributing guide.
